recipes/whatlies/meta.yaml

```diff
@@ -0,0 +1,357 @@
+        # base_packages
+        - altair {{ libver_altair }}
+        - bpemb {{ libver_bpemb }}
+        - gensim {{ libver_gensim }}
+        - matplotlib-base {{ libver_matplotlib_base }}
+        - scikit-learn {{ libver_scikit_learn }}
+      run_constrained:
+        # umap_packages
+        - umap-learn {{ libver_umap_learn }}
+        # rasa_packages
+        - rasa {{ libver_rasa }}
+        # fasttext_packages
+        - fasttext {{ libver_fasttext }}
+        # spacy_packages
+        - spacy {{ libver_spacy }}
+        - spacy-lookups-data {{ libver_spacy_lookups_data }}
+        # s2v_packages (+= spacy_packages)
+        - sense2vec {{ libver_sense2vec }}
+        # tf_packages
+        - tensorflow {{ libver_tensorflow }}
+        - tensorflow-text {{ libver_tensorflow_text }}
+        - tensorflow-hub {{ libver_tensorflow_hub }}
+        # transformers_dep
+        - transformers {{ libver_transformers }}
+        # sentence_tfm_dep
+        - sentence-transformers {{ libver_sentence_transformers }}
+    test:
+      imports:
+        - {{ name_ }}
+      commands:
+        - pip check
+      requires:
+        - pip
+
+  ### umap_packages
+  {% if use_umap_packages is defined %}
+  {% if use_umap_packages or use_all_subpackages %}
+  {% with subpackage_id = "umap" %}
+  - name: {{ name_ }}-{{ subpackage_id }}
+    build:
+      noarch: generic
+      string: {{ build_num }}_{{ subpackage_id }}_extra
+    requirements:
+      run:
+        # RECIPE: 
+        # "umap": base_packages + umap_packages
+        - {{ name_ }} {{ version }}
+        - umap-learn {{ libver_umap_learn }}
+    test:
+      imports:
+        - {{ name_ }}
+      commands:
+        - pip check
+      requires:
+        - pip
+  {% endwith %}
+  {% endif %}
+  {% endif %}
+  
+  ### rasa_packages
+  {% if use_rasa_packages is defined %}
+  {% if use_rasa_packages or use_all_subpackages %}
+  {% with subpackage_id = "rasa" %}
+  - name: {{ name_ }}-{{ subpackage_id }}
+    build:
+      ## NOTE: rasa is not available on conda-forge
+      skip: True  # [win or unix or osx]
+      noarch: generic
+      string: {{ build_num }}_{{ subpackage_id }}_extra
+    requirements:
+      run:
+        # RECIPE: 
+        # "rasa": base_packages + rasa_packages
+        - {{ name_ }} {{ version }}
+      run_constrained:
+        # using run_constrained temporarily
+        # until rasa is made available on
+        # conda-forge channel.
+        - rasa {{ libver_rasa }}
+    test:
+      imports:
+        - {{ name_ }}
+      commands:
+        - pip check
+      requires:
+        - pip
+  {% endwith %}
+  {% endif %}
+  {% endif %}
+  
+  ### fasttext_packages
+  {% if use_fasttext_packages is defined %}
+  {% if use_fasttext_packages or use_all_subpackages %}
+  {% with subpackage_id = "fasttext" %}
+  - name: {{ name_ }}-{{ subpackage_id }}
+    build:
+      ## NOTE: fasttext in NOT available for windows on conda-forge
+      skip: True  # [win]
```

Comment:
You cannot use `skip` when building noarch packages.
```suggestion
```

Ref https://github.com/conda-forge/staged-recipes/pull/16963#issuecomment-974666530

Reply:
Done.